File: app_socket/app_socket/GaiaWatcher/Classes/Io.cs
using System;
using System.Collections.Concurrent;
using System.Collections.Generic;
using System.Linq;
using System.Net.Sockets;
using System.Text;

namespace GaiaWatcher {
    public class Io {
        public int speed {
            get;
            set;
        }
        public int runtime {
            get;
            set;
        }
        public int odo {
            get;
            set;
        }
        public int acc {
            get;
            set;
        }
        public int sos {
            get;
            set;
        }
        public int epc {
            get;
            set;
        }
        public int batt {
            get;
            set;
        }
        public int vcc {
            get;
            set;
        }

        public int accel {
            get;
            set;
        }
        public int decel {
            get;
            set;
        }

        public int tow {
            get;
            set;
        }

        public int motion {
            get;
            set;
        }
        public int fuel {
            get;
            set;
        }

        public int rpm {
            get;
            set;
        }

        public int alarm {
            get;
            set;
        }
        public int mode {
            get;
            set;
        }
        public string ibutton {
            get;
            set;
        }

        public string picture {
            get;
            set;
        }
        public int weight {
            get;
            set;
        }
        public int relay1 {
            get;
            set;
        }
        public int relay2 {
            get;
            set;
        }
        public int relay3 {
            get;
            set;
        }
        public int relay4 {
            get;
            set;
        }
    }
}

File: app_api/app_api_flight/app/model/class/io.php
<?php 
/*
	Created by 		:		Rhalf Wendel D Caacbay
	Created on 		:		20170430

	Modified by 	:		#
	Modified on 	:		#

	functions 		:		Class definition of io.
*/
class Io {
	public $speed;
	public $runtime;
	public $odo;
	public $acc;
	public $sos;
	public $epc;
	public $batt;
	public $vcc;
	public $accel;
	public $decel;
	public $tow;
	public $motion;
	public $fuel;
	public $rpm;
	public $alarm;
	public $mode;
	public $pic;
	public $ibutton;
	public $weight;
	public $relay1;
	public $relay2;
	public $relay3;
	public $relay4;


	

	public function __construct(
		$speed, $runtime, $odo, $acc, $sos,	$epc,
		$batt, $vcc, $accel, $decel, $tow, $motion,
		$fuel, $rpm, $alarm, $mode,	$pic, $ibutton,
		$weight, $relay1, $relay2, $relay3,	$relay4	) {

		$this->speed = $speed;
		$this->runtime = $runtime;
		$this->odo = $odo;
		$this->acc = $acc;
		$this->sos = $sos;
		$this->epc = $epc;
		$this->batt = $batt;
		$this->vcc = $vcc;
		$this->accel = $accel;
		$this->decel = $decel;
		$this->tow = $tow;
		$this->motion = $motion;
		$this->fuel = $fuel;
		$this->rpm = $rpm;
		$this->alarm = $alarm;
		$this->mode = $mode;
		$this->pic = $pic;
		$this->ibutton = $ibutton;
		$this->weight = $weight;
		$this->relay1 = $relay1;
		$this->relay2 = $relay2;
		$this->relay3 = $relay3;
		$this->relay4 = $relay4;
	}

}

?>
